packages/core/src/fonts.css

```diff
@@ -56,8 +50,7 @@
 	font-style: italic;
 	font-weight: 400;
 	src: local('Open Sans Italic'), local('OpenSans-Italic'),
-		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.italic.latin-ext.woff2)
-			format('woff2');
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.italic.latin-ext.woff2) format('woff2');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Quote this URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aquoted-url)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this protocol-relative URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aprotocol-relative-url)